docs/source/install.rst

```diff
@@ -54,12 +53,19 @@ The ROSCO binary file can be copied to your desired folder using:
 
     cp $CONDA_PREFIX/lib/libdiscon.* .
 
+on linux or:
+::
+
+    copy %CONDA_PREFIX%/lib/libdiscon.* .
+
 
-CMake for developers:
-.....................
+on Windows.
+
+
+CMake for developers (Mac/linux)
```

Comment:
If I were using Windows, I would skip right through this section since it says (Mac/Linux). However, the next line has info for Windows users.

Reply:
Good point, that's a typoe. Fixed in one of my rebased commits. I unfortunately can't find the specific commit hash, as I forgot to squash when rebasing and now my commit history is a mess. :(